TestSwitchApi/TestSwitchApi/Models/Request/AddTestSubmissionRequestModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace TestSwitchApi.Models.Request
+{
+    public class AddTestSubmissionRequestModel
+    {
+        [Required]
+        public int TestId { get; set; }
```

Comment:
we never actually check that our model complies with this `[Required]` attribute.
Could you add a check in the controller?

https://docs.microsoft.com/en-us/aspnet/core/mvc/models/validation?view=aspnetcore-3.1

Reply:
Added the checks for the Required Attributes - TestId and TestAnswer